src/clr-angular/emphasis/alert/alert.spec.ts

```diff
@@ -129,12 +136,25 @@ export default function(): void {
       expect(compiled.querySelector('.close').getAttribute('aria-label')).toBe(CLOSE_ARIA_LABEL);
     });
 
-    it('should have an aria-live with value "off"', () => {
-      // https://www.w3.org/TR/wai-aria-1.1/#alert
+    it("should have an aria-live value of polite when you don't apply any attribute", () => {
       const myAlert: HTMLElement = compiled.querySelector('.alert');
+      expect(myAlert.getAttribute('aria-live')).toBe('polite');
+    });
+
+    it('should have an aria-live value of off when apply clrOff', () => {
+      const myAlert: HTMLElement = compiled.querySelector('.alert');
+      fixture.componentInstance.clrOff = true;
+      fixture.detectChanges();
       expect(myAlert.getAttribute('aria-live')).toBe('off');
     });
 
+    it('should have an aria-live value of assertive when apply clrAssertive', () => {
+      const myAlert: HTMLElement = compiled.querySelector('.alert');
+      fixture.componentInstance.clrAssertive = true;
+      fixture.detectChanges();
+      expect(myAlert.getAttribute('aria-live')).toBe('assertive');
+    });
+
```

Comment:
One last bit... we need to test the cascade (Assertive, Off, Polite). Can we add tests that set all three to true and validate that `aria-live` is `assertive` and also a test that sets off and polite to true and tests that `aria-live` is `off`?

One day these tests will save your bacon when somebody comes in here and wrecks your inputs with an enum or decides to make the logic into a one-liner to prove their haxxor skillz.

Reply:
LOL

Reply:
BTW, I had one but did not commit it - thinking this is too much ... My initial test was based on the priority testing them .. ;)